fix(documentation): migration guide (#968)
Vonage/vivid-3: docs/getting-statred/getting-started.md

# Getting Started with Vivid@next

Step-by-step instructions for setting up vivid@next

In case vivid@2.x is installed in your project, make sure you follow the  **[Pre-Installation]**(#Pre-Installation)

---

## install vivid@next

```bash
npm install @vonage/vivid@next
```

---

## Setting the prefix

```js
import { registerDialog } from '@vonage/vivid';

registerDialog ('vwc3');

> ```

Author: why?

# Getting Started with Vivid@3 and up For Vonage Users

> This is a step-by-step instructions for setting up vivid@3 and up.

Reviewer: singular
```suggestion
This is a step-by-step instruction for setting up vivid@3 and up.
```

In case vivid@2.x is installed in your project, make sure you follow the  **[integrating vivid@3 alongside vivid@2]**(#integrating-vivid@3-alongside-vivid@2)

---

## install vivid

```bash
npm install @vonage/vivid
```

---

## Prerequisites

### Tokens

> Add tokens to your app.

Reviewer 2: maybe "choose a *theme* to include" (or something) might make more sense to users who don't necessarily know what *tokens* are?

CSS:

```css
@import "~@vonage/vivid/styles/tokens/theme-light.css";

/* or alternatively */

> @import "~@vonage/vivid/styles/tokens/theme-dark.css";

Reviewer 2: the path should be the one to `node_modules` as for SCSS.
the `~` is not standard, it's only a facility offered by some build tools.

```

SCSS:

```css
@forward 'node_modules/@vonage/vivid/styles/tokens/theme-light.css";
```

In the &lt;head> tag:

```html

> <link rel="stylesheet" href="/assets/styles/tokens/theme-light.css" media="all">

Reviewer 2: this should also target the `node_modules` folder

```

### Font

Vivid uses Montserrat and Roboto Mono Google fonts.

**Vonage** products should use the brand-specified Spezia font families.

> Go to this repo:

Reviewer: To obtain the _Spezia_ webfont kit, go to this repository:

[https://github.com/Vonage/spezia-webfont-kit](https://github.com/Vonage/spezia-webfont-kit)

> Download and add the Spezia font to your app.

Reviewer: Download the font and add it to your project.

> Make sure you add it in the correct path as in the css file (or change the path in the css below):

Reviewer: ```suggestion
Make sure it is added to the correct path as indicated in the CSS file (or update the path in the CSS below).
```

```
assets/fonts/Spezia_Web_Complete/VariableFont/Complete
```

In your css file add this code for setting to font-family:

> Note - the`@import` needs to be declared at the top of the css file.

Reviewer: ```suggestion
Note: The `@font-face` declaration must be placed at the top of the CSS file.
```

	--vvd-typography-heading-3: 500 condensed calc(var(--vvd-size-font-scale-base, 16px) * 1.625)/1.3846153846153846 SpeziaCompleteVariableUpright;
	--vvd-typography-heading-4: 500 condensed calc(var(--vvd-size-font-scale-base, 16px) * 1.25)/1.4 SpeziaCompleteVariableUpright;
	--vvd-typography-base: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 SpeziaCompleteVariableUpright;
	--vvd-typography-base-bold: 600 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 SpeziaCompleteVariableUpright;
	--vvd-typography-base-code: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 SpeziaMonoCompleteVariable;
	--vvd-typography-base-condensed: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 SpeziaCompleteVariableUpright;
	--vvd-typography-base-condensed-bold: 600 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 SpeziaCompleteVariableUpright;
	--vvd-typography-base-extended: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px))/1.5 SpeziaCompleteVariableUpright;

> 	--vvd-typography-base-extended-bold: 600 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px))/1.5 SpeziaCompleteVariableUpright;

Reviewer 2: maybe one or two simpler examples would make this part less intimidating? 😨

Reviewer: generally this could be avoided by simply referring to the https://github.com/vonage/spezia-webfont-kit repo. it's a generous initiative by vivid team which everyone are bound to by brand.
we don't want to be infamous for it

Author: So I'll leaving it as it is

	/* If vivid typography css core style is included in application, setting the '--vvd-size-font-scale-base'
	css variable as derivative will flexibly update font-size by the user preference */
}
```

To add fonts and tokens to the component you need to add the class`vvd-root` to your app.

```html
<body class="vvd-root">...</body>
```

> You can also add it to any wrapping element if you would like to scope the styles to only a certain part of your application.

Reviewer 2: this part is so important that I would put it in its own section before the font explanation, WDYT?

Author: I moved it after typography but gave it a heading to make it more noticeable

# Getting Started with Vivid@next

> Step-by-step instructions for setting up vivid@next

Reviewer: ```suggestion
A step-by-step instructions for getting up and running with vivid@next
```
